auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailFragment.java

```diff
@@ -86,6 +92,16 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         }
     }
 
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        if (!mPasswordInput.isHintAnimationEnabled()) {
```

Comment:
Don't do anything which is not in Trello. Minimal changes please! I wrote it like 5 times already.